app/src/pages/tracing/SpansTable.tsx

```diff
@@ -120,6 +125,11 @@ export function SpansTable(props: SpansTableProps) {
         return <span css={floatRightCSS}>{formatFloat(seconds)}s</span>;
       },
     },
+    {
+      header: "token count (total)",
```

Comment:
<img width="370" alt="Screenshot 2023-08-16 at 6 07 03 PM" src="https://github.com/Arize-ai/phoenix/assets/5640648/0948d26f-96fd-4fae-8376-134645f47bd4">
Interesting that the total is what's important - we might eventually want to break this down like this.